packages/fast-tooling-react/src/css-editor/editor.tsx

```diff
@@ -68,6 +70,12 @@ export default class CSSEditor extends Foundation<
                     return <CSSWidth value={this.props.data} {...config} />;
                 },
             }),
+            new StandardControlPlugin({
+                id: boxShadowPlugInId,
+                control: (config: ControlConfig): React.ReactNode => {
+                    return <CSSBoxShadow value={this.props.data} {...config} />;
```

Comment:
I've just realized this, but this value is not needed, its part of the ControlConfig, so each time you guys are just overriding it. You can remove `value={this.props.data}` in all cases.